lib/js/components/Modal/Modal.vue

```diff
@@ -85,6 +87,7 @@
 	&__content {
 		overflow: auto;
 		padding: $space-s;
+		color: $color-default-text;
```

Comment:
do we have to everride default value defined it https://github.com/bethinkpl/design-system/blob/57a49b3fcc4417303d554b74977bd927d11445a6/lib/styles/design-system.scss#L40 or is it enough to fix this line?

Reply:
If this line
```
color: var($color-default-text); 
```
is not overridden in platform by anything else, it's ok. But I'm aware some parent/rule can set some different color.

Reply:
So what's your suggestion? Do you plan to override default text color in every component?